heareval/predictions/runner.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+"""
+Downstream training, using embeddings as input features and learning
+predictions.
+
+TODO: Add CUDA device.
```

Comment:
This can just be a click option ya?

Reply:
Agreed yeah

Reply:
@jorshi I mean same for task_embedding.py, you should be able to specify the device on the command-line

Reply:
https://github.com/neuralaudio/hear-eval-kit/issues/129